presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -272,6 +272,8 @@
     public static final String NATIVE_EXECUTION_PROGRAM_ARGUMENTS = "native_execution_program_arguments";
     public static final String NATIVE_EXECUTION_PROCESS_REUSE_ENABLED = "native_execution_process_reuse_enabled";
 
+    public static final String INHERIT_NATIVE_EXECUTION_LOG = "inherit_native_execution_log";
```

Comment:
Since we are using a single process for multiple tasks / queries, it is strange to expose this as a session property. If the process has been launched already, setting this property won't have an effect. Am I missing something?

Reply:
Yeah I did find this issue as well. But if you think about it this way: Currently we are also using a session that controls if we want to reuse the process. So there is a valid option that we do not want to reuse the process. Then this session property makes sense.

If say, for example, we decide we do not want to leave reusing of native process as an option, then we need to migrate these native related properties from session to something non-query related.

Reply:
> there is a valid option that we do not want to reuse the process

Why do we have it? Should we review the options we have to make sure all are needed and make sense?